src/workbooks/optimization/workbook.json

```
@@ -113,7 +102,8 @@
             },
             "defaultValue": "value::all",
             "queryType": 1,
-            "resourceType": "microsoft.resourcegraph/resources"
+            "resourceType": "microsoft.resourcegraph/resources",
+            "label": "Resource Group"
```

nit: Microsoft's style guide tells us to use sentence casing.
```suggestion
            "label": "Resource group"
```